tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert not result 
```

Comment:
Great assert! It would also be a good idea to check to make sure the inventory length has not changed because something accidentally got removed.